src/hooks/queries/useGetAfterPartyAttendancesQuery.ts

```
@@ -1,6 +1,6 @@
-import { useQuery } from "@tanstack/react-query";
 import { eventApi } from "@/apis/eventApi";
 import { QueryKey } from "@/constants/queryKey";
+import { useQuery } from "@tanstack/react-query";
```

ESLint

[error] `@tanstack/react-query` import should occur before import of `@/apis/eventApi`

(import/order)